### DISCUSSION TITLE
[Question][Hw0-3] Late submission for homework 0-3 essay

### DISCUSSION THREAD

**Discussion by quanxi1** [prosyslab-classroom/cs424-program-reasoning#284] [Q&A]
Dear TAs and professor, 

I wonder if you apply the same late submission policy for the essay writing homework. 
I heard that it is not graded, but I've just found out that it could be counted as a bonus point in our final score. 
Could you receive late submission(with some penalty if needed) for one or two days? 

Thank you!

**Comment by KihongHeo** [comment]
Hi.
No late submission will be accepted for this homework.
It was previously announced and some students already submitted their articles.
